_config-blog.yml

```diff
@@ -33,4 +33,10 @@ exclude:
   - tests/schema/tags.yml
   - tmp/
   - vendor
+  - _data/digital-identity.yml
+  - _data/digital-wallet.yml
+  - _data/general-glossary.yml
+  - _data/techy-terms.yml
+  - .circleci/make-yaml.py
```

Comment:
Could replace this with a - like `*.py` or `_data/glossary/*.yml` (if you put the files in `_data/glossary`)